dawn/test/integration-test/unstructured/AtlasVerifier.h

```diff
@@ -106,6 +106,28 @@ class AtlasVerifier {
 
     return verified;
   }
+
+  template <typename ValT, typename iteratorT, template <typename> class FieldT>
+  bool compareToylibField(const iteratorT& iter, FieldT<ValT>& lhs, FieldT<ValT>& rhs, size_t kSize,
```

Comment:
Should `AtlasVerifier` become `UnstructuredVerifier` then? Since we are generalizing it would make sense.

Reply:
Absolutely, wanted to do that but forgot. Thx!